src/PyCall.jl

```diff
@@ -708,6 +708,14 @@ for (mime, method) in ((MIME"text/html", "_repr_html_"),
 end
 
 #########################################################################
+# Expose Python docstrings to Julia doc system
+
+if isdefined(Docs, :getdoc)  # Introduced in Julia .6
+    Docs.getdoc(o::PyObject) = Markdown.parse(o[:__doc__])
```

Comment:
For now, however, can't we just return the string?

(Since we know it is supposed to return a string, it would be better to do `String(o["__doc__"])` to skip the type-introspection step.)